lib/destination/ddl/ddl.go

```diff
@@ -137,11 +137,11 @@ func (a AlterTableArgs) buildStatements(cols ...columns.Column) ([]string, []col
 	} else {
 		for _, colSQLPart := range colSQLParts {
 			var sqlQuery string
-			if _, ok := a.Dialect.(sql.MSSQLDialect); ok {
+			if a.Dialect.SupportsColumnKeyword() {
```

Comment:
Can we create an interface called `GetAlterColumnQuery()`?

Reply:
Yep.